src/components/AssetManagementPanel/AssetManagementForm/FormVariants/SurrenderForm/SurrenderForm.tsx

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import { ButtonSolidGreenWhite, ButtonSolidRedWhite, ButtonSolidWhiteGrey } from "../../../../UI/Button";
+import { AssetManagementActions } from "../../../AssetManagementContainer";
+import { LoaderSpinner } from "../../../../UI/Loader";
+import { AssetInformationPanel } from "../../../AssetInformationPanel";
+import { AssetTitle } from "../../../AssetTitle";
+import { AssetManagementTitle } from "../../AssetManagementTitle";
+import { SkeletonPlaceholder } from "../../SkeletonPlaceholder";
+import { EditableAssetTitle } from "./../EditableAssetTitle";
+
+interface SurrenderFormProps {
+  formAction: AssetManagementActions;
+  onSetFormAction: (nextFormAction: AssetManagementActions) => void;
+  tokenId: string;
+  tokenRegistryAddress: string;
+  beneficiary?: string;
+  holder?: string;
+  surrenderingState: string;
+  handleFormAction: () => void;
+}
+
+export const SurrenderForm = ({
+  formAction,
+  onSetFormAction,
+  tokenId,
+  tokenRegistryAddress,
+  beneficiary,
+  holder,
+  surrenderingState,
+  handleFormAction,
+}: SurrenderFormProps) => {
+  const isPendingConfirmation = surrenderingState === "PENDING_CONFIRMATION";
+  const isConfirmed = surrenderingState === "CONFIRMED";
+
+  const onBackHandler = () => {
+    if (isPendingConfirmation) {
+      return false;
+    } else {
+      onSetFormAction(AssetManagementActions.None);
+    }
+  };
+
+  return (
+    <div className="row py-3">
+      <div className="col-12">
+        <AssetManagementTitle onBack={onBackHandler} formAction={formAction} />
+        <div className="row mb-3">
+          <div className="col-12 col-lg">
+            <AssetInformationPanel tokenId={tokenId} tokenRegistryAddress={tokenRegistryAddress} />
+          </div>
+          <div className="col-12 col-lg">
+            {beneficiary ? (
```

Comment:
do we need to assert this anymore? Are the children already handling the nullable state?